### ISSUE TITLE
please consider a possibility to switch off the graphical startup screen by patching the ISO

### ISSUE THREAD

**Issue by Zugschlus** [grml/grml-live#45]
I regularly use a grml USB stick in a server's USB port to have a rescue system that is independent of both local disk and network. Of course, the easiest way to have this is just dd'ing an grml iso to
the medium and then plugging it in without more ado. In the past, I had to build my own grml iso to have a non-graphical startup to be able to select the grml startup option, since that one is usually shown with a fancy graphical isolinux screen and doesn't work if the only thing one has is a serial port.

I would love to have the possibility to do a simple patch in an otherwise unchanged grml image so that my ProLiant grml is as close to "release" grml as possible. I would appreciate a hint if that is
already possible.

While I do understand that having a nice-looking selection screen is nice and important for user acceptance, this severely reduces grml's attractivity for the server jockey who usually doesn't have graphical access to the systems.

A possible implementation would be to include a longish placeholder comment into the isolinux configuration that can be used to patch in additional commands in an existing iso. I am not afraid of a hex editor, so I'm fine with doing that.

Greetings
Marc

**Comment by mika** [comment]
There already is https://github.com/grml/grml-live/blob/master/templates/boot/isolinux/default.cfg - does this help you?

**Comment by Zugschlus** [comment]
The pointer towards the git templates was really helpful. I found a comment in https://github.com/grml/grml-live/blob/master/templates/boot/isolinux/isolinux.cfg:

```
# finally either include 'console.cfg' for console based bootsplash
# or 'vesamenu.cfg' for graphic menu version of bootsplash (adjusted
# automatically in build process via grml-live):
```

with that, I was able to use dhex on the iso image, to remove the # in front of the serial stanza and to replace include vesamenu.cfg by include console.cfg.

This is exactly what I need.

I would however suggest putting in placeholders as in default.cfg also in isolinux.cfg.

**Comment by Zugschlus** [comment]
I have put instructions in the wiki, http://wiki.grml.org/doku.php?id=rescue
The issue is now reduced to "put in palceholers into isolinux.cfg" and can be closed then. Thanks for helping.

**Comment by Zugschlus** [comment]
closed by merging PR #51
